src/main/java/uk/gov/hmcts/reform/profilesync/service/impl/ProfileUpdateServiceImpl.java

```diff
@@ -77,7 +80,8 @@ private void syncUser(String bearerToken, String s2sToken,
         log.info("Body response::" + response.body().toString());
         if (response.status() > 300) {
 
-            log.error("Exception occurred while updating the user profile: Status - {}" + userId + ":" + updatedUserProfile.getIdamStatus());
+            log.error("Exception occurred while updating the user profile: Status - {}" + userId + ":"
```

Comment:
Please remove + I think with {} no need +. Use below

 log.error("Exception occurred while updating the user profile: Status - {}" ,userId);

Reply:
Done